packages/slate-editor/src/extensions/blink-nodes/BlinkNodesExtension.tsx

```diff
@@ -0,0 +1,20 @@
+import type { Extension } from '@prezly/slate-commons';
+
+export function BlinkNodesExtension(): Extension {
```

Comment:
IMO "blink" is not precisely what it does. It's more like "highlight". 

Blinking is when it goes on and off, repeated. Not once.

Reply:
what about "Flash" ?